subprojects/xcfa/xcfa-cli/src/test/java/hu/bme/mit/theta/xcfa/cli/XcfaCliVerifyTest.kt

```diff
@@ -78,8 +77,8 @@ class XcfaCliVerifyTest {
         fun chcFiles(): Stream<Arguments> {
             return Stream.of(
                 Arguments.of("/chc/chc-LIA-Lin_000.smt2", ChcFrontend.ChcTransformation.FORWARD, "--domain PRED_CART"),
-                Arguments.of("/chc/chc-LIA-Arrays_000.smt2", ChcFrontend.ChcTransformation.BACKWARD,
-                    "--domain PRED_CART"),
+//                Arguments.of("/chc/chc-LIA-Arrays_000.smt2", ChcFrontend.ChcTransformation.BACKWARD,
```

Comment:
This test becomes stuck with the default ARG equality implementation.

Reply:
clarification: _stuck_ as in not progressing until running out of heap space, not as in the progress check-related _stuck_.

Reply:
I try to debug this issue: I commented my first findings in the teams chat (it is muted by default, so you may have not noticed it).